blog/_posts/2022-11-04-scala-3.2.1-released.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.2.1 released!
+---
+
+[Scala 3.2.1](https://github.com/lampepfl/dotty/releases/tag/3.2.1) is now available!
+
+This version is a bug-fixing release focusing mainly but not exclusively on solving rare issues with code not compiling after migration to 3.2.0. This release also brings two notable enhancements:
```

Comment:
`migration` seems like a big word here giving a false bad impression that bumping the compiler from `3.1` to `3.2` requires a lot of work